### ISSUE TITLE
Please add fixed width font (was: Removing button status bar alters 'AUTO' sizing)

### ISSUE THREAD

**Issue by istnv** [bitfocus/companion#1533]
The `AUTO` text size option mis-calculates text size when using the `Remove the topbar on each button` option.

Example: create a new regular button, add `00:00` for the button text.
![image](https://user-images.githubusercontent.com/51003420/112883447-28874000-9083-11eb-9b4c-b625971779f7.png)
![image](https://user-images.githubusercontent.com/51003420/112883495-389f1f80-9083-11eb-8a56-316d2f9a2c16.png)

**Comment by Julusian** [comment]
I agree that this is not desirable, but technically it is working as intended.
`:` is considered a safe point to break for over lines, and by removing the top bar there is room for two lines of the larger text.

To do anything here is going to get complex, and I don't know what the rule should be.

Because of this I am converting this to an enhancement task

**Comment by istnv** [comment]
I would really like a font that had fixed-width numbers. It is really annoying when displaying a countdown timer and HH:MM bounces back and forth from single to double line depending on how many '1's in the timer.

**Comment by Julusian** [comment]
I often up changing the font size away from auto, to avoid issues like that. As well as to make a group of buttons all match.

**Comment by istnv** [comment]
A fixed size works fine until you use a dynamic timer that drops off hours and minutes when those 'digits' become zero. It was intended to draw attention to the count being almost over as the numbers would get slightly bigger on the button. Bouncing sizes between certain seconds is too much attention :) It was not an issue until the extra pixels became available. Hmmm. A variable->trigger action to force font size? I don't see a way to do that, either.

**Comment by Julusian** [comment]
hmm.. I was about to suggest that making the break characters configurable would be simple, but that doesnt appear to help.
![image](https://user-images.githubusercontent.com/1327476/122640483-be1cf380-d0f7-11eb-89fe-8c57cb2b2ce4.png)
Even long strings will break after a certain point

**Comment by istnv** [comment]
The image you posted _should_ fit completely at **14pt**. It does fit **18pt** with ':' separators. 
The size-down or split text choice works visually different with the 14 extra pixels. I'll look around there (when I find it)

**Comment by dnmeid** [comment]
I put this in the graphics overhaul projet for the request of a fixed width font.

**Comment by AJolly** [comment]
Would ge nice if we could have different font sizes for different parts of the button text (or if auto font size worked on a per line basis)
